docs/quickstart/aws.md

```diff
@@ -0,0 +1,152 @@
+## Requirements 
+
+This quickstart assumes you have access to admin level access to AWS. 
+
+## Setting Up Security Groups for EC2
+
+To set up a security group for an EC2 instance, navigate to the AWS EC2 Dashboard. In the side panel click `Security Groups` .
+
+In the upper right hand corner, click `Create Security Group` . In the creator screen, name the security group something relevant to the EC2 instance you will create.
+
+<img src="../images/quickstart/aws/aws1.png">
+
+For ease of use, we are going to set this up using the simplest rules.
+
+<img src="../images/quickstart/aws/aws2.png">
+
+Create a new `Inbound Rule` that allows for SSH from your computer’s IP address.
+
+You’ve now created a security group that will be used by your EC2 instance.
+
+## Setting Up Your EC2 instance
+
+On the EC2 dashboard, click `Instances`. This will take you to all the EC2 instances you have created. Click `Launch New Instance`. Name the EC2 instance following the naming convention of your choice.
```

Comment:
We should probably mention that we're creating the instance for the coder server, not the workspaces. I'm thinking about the naive user that clicks to this Quickstart from the website.